app/jobs/pusher_activity_notification_job.rb

```diff
@@ -7,21 +7,21 @@ def perform(recipient, activity)
     Pusher.trigger(
       'telephone_appointment_planner',
       "guider_activity_#{recipient.id}",
-      body: render_activity(activity, true)
+      body: render_activity(activity, true, true)
```

Comment:
Passing booleans into methods like this is generally not a good idea, unless the method name makes it obvious what the boolean is for. Something like this would be more readable:

```
render_activity(details: false)

def render_activitiy(activity, options={})
  options = {details: true}.merge(options)
end
```

And now you can have details default to true, so that you only need to pass in options when you want.